3rdParty/fuerte/include/fuerte/connection.h

```diff
@@ -69,14 +69,14 @@ class Connection : public std::enable_shared_from_this<Connection> {
   /// When a response is received or an error occurs, the corresponding
   /// callbackis called. The callback is executed on a specific
   /// IO-Thread for this connection.
-  virtual MessageID sendRequest(std::unique_ptr<Request> r,
-                                RequestCallback cb) = 0;
+  virtual void sendRequest(std::unique_ptr<Request> r,
```

Comment:
What happens if the CB is never called? Do you still have way to clean up without the id? (Note to myself: check if CB has timeout)